components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -302,6 +304,54 @@ void EnsureRewardsBaseDirectoryExists(const base::FilePath& path) {
     base::CreateDirectory(path);
 }
 
+net::NetworkTrafficAnnotationTag
+GetNetworkTrafficAnnotationTagForFaviconFetch() {
+  return net::DefineNetworkTrafficAnnotation(
+      "brave_rewards_favicon_fetcher", R"(
+      semantics {
+        sender:
+          "Brave Rewards Media Fetcher"
+        description:
+          "Fetches favicon for media publishers in Rewards."
+        trigger:
+          "User visits a media publisher content."
+        data: "Favicon for media publisher."
+        destination: WEBSITE
+      }
+      policy {
+        cookies_allowed: NO
+        setting:
+          "This feature cannot be disabled by settings."
+        policy_exception_justification:
+          "Not implemented."
+      })");
+}
+
+net::NetworkTrafficAnnotationTag GetNetworkTrafficAnnotationTagForURLLoad() {
+  return net::DefineNetworkTrafficAnnotation("rewards_service_impl", R"(
+      semantics {
+        sender: "Brave Rewards service"
+        description:
+          "This service lets users anonymously support the sites they visit by "
+          "tallying the attention spent on visited sites and dividing up a "
+          "monthly BAT contribution."
+        trigger:
+          "User-initiated for direct tipping or on a set interval while Brave "
+          "is running for monthly contributions."
+        data:
+          "Publisher and contribution data."
+        destination: WEBSITE
+      }
+      policy {
+        cookies_allowed: NO
+        setting:
+          "You can enable or disable this feature via the Brave Shields icon "
```

Comment:
`You can enable or disable this feature via the Brave Shields icon` -> `You can enable or disable this feature via the BAT icon in the URL bar`

Reply:
Done. thanks!